packages/manager/apps/pci-rancher/src/components/Table/NumberCell/NumberCell.component.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import { Cell } from '@tanstack/react-table';
+import { RancherService } from '@/types/api.type';
+import '../Table.scss';
+
+interface DisplayCellInterface {
+  cell: Cell<RancherService, number>;
+}
+
+function DisplayCellNumber({ cell }: Readonly<DisplayCellInterface>) {
+  const number = cell.renderValue();
+  return <OsdsText color={ODS_THEME_COLOR_INTENT.text}>{number}</OsdsText>;
+}
+
+export default DisplayCellNumber;
```

Comment:
I think you should name your component the same as your file (or the opposite)

Reply:
i was inspired by a Text Component and for consistency reason i will not change the name of this file.